ui/ozone/platform/starboard/gl_ozone_egl_starboard.cc

```diff
@@ -0,0 +1,100 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "ui/ozone/platform/starboard/gl_ozone_egl_starboard.h"
+
+#include <memory>
+
+#include "starboard/egl.h"
+#include "starboard/window.h"
+#include "ui/gfx/vsync_provider.h"
+
+namespace {
+// TODO(b/371272304): Currently targeting a fixed 60fps. Make this dynamic based
+// on device capabilities.
+base::TimeDelta GetVSyncInterval() {
+  return base::Seconds(1) / 59.94;
```

Comment:
This is not 60Hz like l.24 says (display refresh rates come in Hz, not in fps).

Reply:
Done.